arbiter/arbiter-core/src/main/java/org/deeplearning4j/arbiter/optimize/generator/genetic/population/PopulationModel.java

```diff
@@ -0,0 +1,188 @@
+/*******************************************************************************
+ * Copyright (c) 2015-2018 Skymind, Inc.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package org.deeplearning4j.arbiter.optimize.generator.genetic.population;
+
+import org.deeplearning4j.arbiter.optimize.generator.genetic.Chromosome;
+import org.deeplearning4j.arbiter.optimize.generator.genetic.culling.CullOperator;
+import org.deeplearning4j.arbiter.optimize.generator.genetic.culling.LeastFitCullOperator;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * The population model handles all aspects of the population (initialization, additions and culling)
+ *
+ * @author Alexandre Boulanger
+ */
+public class PopulationModel {
+    private static final int DEFAULT_POPULATION_SIZE = 30;
+
+    /**
+     * A comparator used when higher fitness value is better
+     */
+    public class MaximizeScoreComparator implements Comparator<Chromosome> {
+        @Override
+        public int compare(Chromosome lhs, Chromosome rhs) {
+            if (lhs.fitness < rhs.fitness)
+                return 1;
+            else if (rhs.fitness < lhs.fitness)
+                return -1;
+            return 0;
```

Comment:
Let's replace these 5 lines with: ```return Double.compareDouble(lhs.getFitness(), rhs.getFitness())``` (possibly with order switched, if that's the intention here).
Note I'm assuming the fitness field is protected/private and getter via ```@Data``` was added, as per earlier review comment.
Same thing for other comparator.